server/container/guice/guice-common/src/test/java/org/apache/james/MailsShouldBeWellReceived.java

```diff
@@ -40,6 +46,7 @@
     ConditionFactory CALMLY_AWAIT = Awaitility
         .with().pollInterval(Duration.ONE_HUNDRED_MILLISECONDS)
         .and().pollDelay(Duration.ONE_HUNDRED_MILLISECONDS)
+        .timeout(Duration.FIVE_MINUTES)
```

Comment:
five minutes timeout may be too much for most of the tests

Reply:
Yes indeed but i prefer to have a worst scenario of 5 minutes than another flaky test.

Reply:
What I mean is: you changed the timeout for existing tests (that, at the very least, should not be part of this commit).
Please apply this timeout only to the test you wrote.